src/services/dashboards/modules/DashboardCloneModal.vue

```diff
@@ -121,52 +122,60 @@ export default defineComponent<Props>({
         const state = reactive({
             proxyVisible: props.visible,
-            filteredVisibilityList: computed(() => (props.manageDisabled ? visibilityList.filter((item) => item.name === DASHBOARD_PRIVACY_TYPE.USER) : visibilityList)),
+            filteredVisibilityList: computed(() => visibilityList),
+            isProjectDashboard: computed(() => Object.prototype.hasOwnProperty.call(props.dashboard, 'project_id')),
+            dashboardNameList: computed<string[]>(() => {
+                if (state.isProjectDashboard) {
+                    return store.state.dashboard.projectItems
+                        .filter((item) => (item.project_id === (props.dashboard as ProjectDashboardModel).project_id))
+                        .map((_item) => {
+                            if (_item.name !== props.dashboard?.name) return _item.name;
+                            return '';
+                        });
+                }
+                return store.state.dashboard.domainItems.map((item) => {
+                    if (item.name !== props.dashboard?.name) return item.name;
+                    return '';
+                });
+            }),
         });
-        // const _invalid_unique = 'Dashboard name must be unique'; i18n.t('DASHBOARDS.FORM.VALIDATION_DASHBOARD_NAME_UNIQUE')
-        // const _invalid_input = 'Please input dashboard name'; i18n.t('DASHBOARDS.FORM.VALIDATION_DASHBOARD_NAME_INPUT')
 
         const handleUpdateVisible = (visible) => {
             state.proxyVisible = visible;
             emit('update:visible', visible);
         };
 
-        // const getCustomLayouts = async () => {
-        //     const hasDefaultId = Object.prototype.hasOwnProperty.call(props.dashboard, 'default_layout_id');
-        //     if (hasDefaultId && (!props.dashboard.custom_layouts || props.dashboard.custom_layouts.length === 0)) {
-        //         return await fetchDefaultLayoutData(props.dashboard.default_layout_id as string) as CustomLayout[];
-        //     }
-        //     return props.dashboard.custom_layouts as CustomLayout[];
-        // };
-
-        // const makeDashboardCreateParam = async (): Promise<DashboardCreateParam> => ({
-        //     name: name.value,
-        //     custom_layouts: await getCustomLayouts(),
-        //     period_type: props.dashboard.period_type as PeriodType ?? PERIOD_TYPE.AUTO,
-        //     period: props.dashboard.period,
-        // });
-
-        const createPublicDashboard = async (): Promise<string|undefined> => {
+        const createDomainDashboard = async (): Promise<string|undefined> => {
             try {
-                // const res = await SpaceConnector.client.costAnalysis.publicDashboard.create(await makeDashboardCreateParam() as DashboardCreateParam);
-                // return res.public_dashboard_id;
+                const res = await SpaceConnector.clientV2.dashboard.domainDashboard.create({
+                    // viewers: viewers.value
+                });
```

Comment:
question: is it ok to create without any options?

Reply:
It should be added ! 
I did not checked api working.
I'll add some annotations 
Thank you ~